packages/ui-components/src/icon/widgets/tabbarsvg.ts

```diff
@@ -17,9 +18,16 @@ export class TabBarSvg<T> extends TabBar<T> {
    *
    * @param options - The options for initializing the tab bar.
    */
-  constructor(options: TabBar.IOptions<T> = {}) {
+  constructor(options: TabBarSvg.IOptions = {}) {
     options.renderer = options.renderer || TabBarSvg.defaultRenderer;
     super(options);
+    const trans = ((options && options.translator) || nullTranslator).load(
+      'jupyterlab'
+    );
+    addIcon.element({
+      container: this.addButtonNode,
+      title: trans.__('New Launcher')
```

Comment:
On Firefox the working is `Open a new tab`. I think we should also be more explicit here.

```suggestion
      title: trans.__('Open a New Launcher')
```

I don't know if it is better to say _a new launcher_ or _a new tab_. The latter is probably more common and easily to be understood by the users.

Reply:
Safari and Chrome only have `New Tab` so I personally think simpler is better. I'm ok switching to tab instead of launcher, but one of the reasons I used this exact text is it's the tool tip for the current button in the file browser and should already be translated.